train.py

```diff
@@ -252,8 +256,12 @@ def main():
 
             if step % 50 == 0:
                 save(saver, sess, logdir, step)
+                last_save_step = step
 
     finally:
+        if step > last_save_step:
+            print()
```

Comment:
Great, thanks! For next time, GitHub squashes automatically when merging and gives that cool "commited with" message rather than creating a merge commit, so no need for that 😄